orders/tests.py

```diff
@@ -0,0 +1,79 @@
+from django.utils import timezone
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+from .models import Order
+from .models import StatusOptions
+import pytest
+
+STARTING_DATE = timezone.now()
+FINISHING_DATE = timezone.now()
+TOTAL_PRICE = 100
+STATUS = StatusOptions.Pending
+
+
+@pytest.fixture
+def order():
+    return Order(dog_owner_id=DogOwner.objects.get(pk=1),
+                 daycare_id=DayCare.objects.get(pk=1),
+                 starting_date=STARTING_DATE,
+                 finishing_date=FINISHING_DATE,
+                 total_price=TOTAL_PRICE,
+                 )
+
+
+@pytest.mark.django_db
+class TestOrderModel:
+    def test_create_order(self, order):
+        new_order = Order.create(dog_owner_id=order.dog_owner_id,
+                                 daycare_id=order.daycare_id,
+                                 starting_date=order.starting_date,
+                                 finishing_date=order.finishing_date,
+                                 total_price=order.total_price)
+        assert new_order in Order.objects.all()
+
+    def test_delete_order(self, order):
+        order_to_delete = Order.create(dog_owner_id=order.dog_owner_id,
+                                       daycare_id=order.daycare_id,
+                                       starting_date=order.starting_date,
+                                       finishing_date=order.finishing_date,
+                                       total_price=order.total_price)
+
+        order_to_delete.delete()
+        assert order_to_delete not in Order.objects.all()
+
+    def test_get_all_orders_by_dog_owner_id(self, order):
+        dog_owner_orders = Order.get_all_dog_owner_orders(order.dog_owner_id)
+
+        print(dog_owner_orders)
```

Comment:
I don't understand what are we testing here, 
It seems like you only print something, maybe we print wrong data?

Reply:
You're right, it tests that the function do pull dog owner's orders from the database.
Maybe the check should be different, like asking for each order if it's in the database or something.
I'll change it.